buyer-api/src/routes/dataOrders/closeDataOrder.js

```diff
@@ -37,20 +38,17 @@ const router = express.Router();
  *         description: Problem on our side
  */
 router.post(
-  '/:orderAddress/end',
-  validateAddress('orderAddress'),
+  '/:id/close',
+  fetchDataOrder,
   asyncError(async (req, res) => {
-    const { orderAddress } = req.params;
+    if (req.dataOrder.status !== 'created') {
```

Comment:
I'd move this check into the operation since it belongs to our business.

Reply:
As this is not yet discussed, let's leave it like this for now. We'll later come back to this.